rootfs/scheduler/__init__.py

```diff
@@ -1049,6 +1051,44 @@ def _healthcheck(self, controller, routable=False, path='/', port=5000, delay=30
 
         return controller
 
+    def _readiness_probe(self, controller, delay=30, timeout=5, period_seconds=5,
```

Comment:
Maybe we should name it differently and add in more description about it being there *only* when a healthcheck is not provided and only when it is a build pack application